tools/i18n/update.js

```diff
@@ -69,22 +72,30 @@ htmlFiles.forEach(function (htmlFile) {
         return previous;
     }, {});
 
-    updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
+    return updateLanguageXlfJson(sourceHtmlRelativePath, strings).then(function (result) {
```

Comment:
I would prefer if we can put `then`, `fail` etc callbacks in a new line, to improve readability. That was an implicit convention that we used to follow.
```
return updateLanguageXlfJson(/* ... */)
    .then(function () { 
        // implementation here
    });

```